client/Navigation.js

```diff
@@ -28,13 +27,13 @@ export default class Navigation extends Component {
 					</ul>
 				) }
 			</li>
-		)
+		);
 	}
 
 	render() {
 		// @todo This should be updated to use a wp data store.
-		const items = window.wcNavigation || [];
-		
+		const items = window.wcSettings.wcNavigation || [];
```

Comment:
Just to be extra safe here, should we check to make sure `window.wcSettings` exists?  Something like:

```suggestion
		const items = window.wcSettings && window.wcSettings.wcNavigation ? window.wcSettings.wcNavigation : [];
```